conf/pipeline/sarek/cfc.config

```diff
@@ -0,0 +1,24 @@
+// Profile config names for nf-core/configs
+
+params {
+  // Specific nf-core/configs params
+  config_profile_contact = 'Friederike Hanssen (@FriederikeHanssen)'
+  config_profile_description = 'nf-core/sarek CFC profile provided by nf-core/configs'
+}
+
+// Specific nf-core/sarek process configuration
+process {
+  withName:'StrelkaSingle|Strelka|StrelkaBP|MantaSingle|Manta' {
+    memory = 128.GB
+    cpus = 16
+  }
+  withName:'MSIsensor_scan|MSIsensor_msi' {
+    memory = 128.GB
+  }
+  withName:BamQC {
+    memory = 372.GB
+  }
+  withName:MapReads{
+    cpus = 48
```

Comment:
Also here this will go to qbic queue. Which is fine for the Mapping step I guess. But then also more memory could be used to just 60.GB?

Reply:
so far we haven't had issues with 'just' using 60GB memory. Looking at the execution_reports for my large data sets, I supplied 372GB but then only 120GB or so were used. I don't think it makes sense with those numbers to go over 128Gb or so. (Could change for BWAMem2, that seems to scale better with increased resource, but that is a thing for the next release).

Reply:
ok if "just" 60GB are needed then we can use again the `compute` queue, where nodes have 20 cpus. Would that be ok?